docs/examples/getDataMoverResources.go

```diff
@@ -0,0 +1,217 @@
+	fmt.Printf("Processing: %v\n", getNumberOfMatches(batchingLines, "Processing"))
+	fmt.Printf("Queued: %v\n\n", getNumberOfMatches(batchingLines, "Queued"))
+
+	number = getNumberOfResources(captureShellOutput("oc get replicationsource -A"))
+	fmt.Printf("Total ReplicationSources: %v\n", number)
+	if details {
+		volumeSnapshotContentDetails()
+		replicationSourceDetails()
+	}
+}
+
+func restoreSummary(details bool) {
+	// TODO
+	// echo -e "Get Restores:\n"
+	// oc -n openshift-adp exec deployment/velero -c velero -it -- ./velero get restore ;
+	// echo -e "\nTotal VSR: " `oc get vsr -A | sed 1d | wc -l` ;
+	// echo "Completed: " `oc get vsr -A | grep -c Completed` ;
+	// echo "InProgress: " `oc get vsr -A | grep -c InProgress` ;
+	// echo "SnapshotRestoreDone: " `oc get vsr -A | grep -c SnapshotRestoreDone` ;
+	// echo -e "\nVSR STATUS" ;
+	// echo "Completed: " `oc get vsr -A -oyaml | grep batching | grep -c Completed` ;
+	// echo "Processing: " `oc get vsr -A -oyaml | grep batching | grep -c Processing` ;
+	// echo "Queued: " `oc get vsr -A -oyaml | grep batching | grep -c Queued` ;
+	// echo -e "\nTotal ReplicationDestinations: " `oc get replicationdestination -A | sed 1d | wc -l` ;
+	if details {
+		volumeSnapshotContentDetails()
+		replicationDestinationDetails()
+	}
+}
+
+func volumeSnapshotContentDetails() {
+	fmt.Println("\n***** VOLUME SNAPSHOT CONTENT ******")
+	output := captureShellOutput("oc get vsc -A -o=custom-columns=:.metadata.name")
+	// TODO check if removing spaces did not brake anything
+	command := `oc get vsc %v -o jsonpath='{"Name:"}{@.metadata.name}{"ReadyToUse:"}{@.status.readyToUse}{"creationTime:"}{@.metadata.creationTimestamp}'`
+	for _, vsc := range strings.Split(output, "\n") {
+		// TODO | column -t
+		if len(strings.Trim(vsc, " \n")) > 0 {
+			output = captureShellOutput(fmt.Sprintf(command, vsc))
+			fmt.Println(output)
+		}
+	}
+}
+
+func replicationSourceDetails() {
+	// TODO
+	// echo -e "\n***** REPLICATION SOURCE ******"
+	// for i in $(oc -n openshift-adp get replicationsource -A -o=custom-columns=:.metadata.name); do
+	//     oc -n openshift-adp get replicationsource $i -o jsonpath='{"Name: "}{@.metadata.name}{" SyncDuration: "}{@.status.lastSyncDuration}{" sourcePVC: " }{@.spec.sourcePVC}' | column -t
+	// done
+}
+
+func replicationDestinationDetails() {
+	// TODO
+	// echo -e "\n***** REPLICATION DESTINATION ******"
+	// for i in $(oc -n openshift-adp get replicationdestination -A -o=custom-columns=:.metadata.name); do
+	//     oc -n openshift-adp get replicationdestination $i -o jsonpath='{"Name: "}{@.metadata.name}{" SyncDuration: "}{@.status.lastSyncDuration}{" sourcePVC: " }{@.spec.sourcePVC}' | column -t
+	// done
+}
+
+func init() {
+	rootCmd.Flags().BoolVarP(&backup, "backup", "b", false, "Check a backup")
+	rootCmd.Flags().BoolVarP(&restore, "restore", "r", false, "Check a restore")
+	rootCmd.MarkFlagsMutuallyExclusive("backup", "restore")
+	// https://github.com/spf13/cobra/pull/1952
```

Comment:
backup and restore should be required, but there is not this feature in cobra yet (but it is already merged)